doc/source/whatsnew/v1.1.0.rst

```diff
@@ -36,6 +36,55 @@ For example:
    ser["2014"]
    ser.loc["May 2015"]
 
+.. _whatsnew_110.grouper_resample_origin:
+
+Grouper and resample now supports the arguments origin and offset
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+:class:`Grouper` and :class:`DataFrame.resample` now supports the argument `origin`. The timestamp on which to adjust the grouping. (:issue:`31809`)
+
+The bins of the grouping are adjusted based on the beginning of the day of the time series starting point. This works well with frequencies that are multiples of a day (like `30D`) or that divides a day (like `90s` or `1min`). But it can create inconsistencies with some frequencies that do not meet this criteria. To change this behavior you can now specify a fixed timestamp with the argument ``origin``.
+
+For example:
+
+.. ipython:: python
+
+    start, end = "1/10/2000 02:00:00", "1/20/2000 02:00"
+    middle = "1/15/2000 02:00"
+    rng = pd.date_range(start, end, freq="1231min")
+    ts = pd.Series(np.arange(len(rng)) * 3, index=rng)
```

Comment:
make this several blocks its *very* hard to grok this

Reply:
You are right... I tried to make this clearer in the new changes. Tell me if that's okay for you.